server/obsolete/async_socket.py

```diff
@@ -0,0 +1,74 @@
+import asyncore
+import logging
+import socket
+from time import sleep
+
+
+class FlaskClientSocketASYNC(asyncore.dispatcher_with_send):
```

Comment:
Just curious, the plans for this class?

Reply:
Was exploring other socket modules that have async functionality so nothing is blocked while reading/sending.
Currently not used, but may be used for CommunicationRouter if that's happening

Kept code in /obsolete folder to store not currently used scripts and scratch files